js/main.js

```diff
@@ -1,2 +1,87 @@
 /* global data */
 /* exported data */
+// SWITCHING PAGES
+var buttonEntry = document.querySelector('.view-button');
+var buttonNew = document.querySelector('.new');
+
+var enterElement = document.getElementById('test1');
+var viewElement = document.getElementById('test2');
+
+buttonEntry.addEventListener('click', viewClick);
+buttonNew.addEventListener('click', entryClick);
+
+function viewClick(event) {
+  window.location.reload(); // Refresh page
+  enterElement.className = 'view hidden';
+  viewElement.className = 'view';
+}
+
+function entryClick(event) {
+  enterElement.className = 'view';
+  viewElement.className = 'view hidden';
+}
+
+// CREATE ENTRY
+var results = document.querySelector('#entry-form');
+var inputs = document.querySelector('#entry-form').elements;
+results.addEventListener('submit', submitValues);
+
+var pic = document.querySelector('#entry-form img');
+var photo = document.querySelector('#entry-form input[name=\'photoUrl\']');
+var picUpdate = document.querySelector('img');
+
+photo.addEventListener('input', addPhoto);
+
+function addPhoto(event) {
+  picUpdate.src = event.target.value;
+}
+
+function submitValues(event) {
+  event.preventDefault();
+  var obj = {};
+  obj.title = inputs[0].value;
+  obj.photoUrl = inputs[1].value;
+  obj.notes = inputs[2].value;
+
+  obj.entryID = data.nextEntryId;
+
+  data.nextEntryId += 1;
+
+  data.entries.unshift(obj);
+
+  pic.src = 'images/placeholder-image-square.jpg';
+  document.querySelector('#entry-form').reset();
+
+  enterElement.className = 'view hidden';
+  viewElement.className = 'view';
+  window.location.reload(); // Refresh page
```

Comment:
you don't ever want to reload the app during runtime. Instead, you want to call the render function noted in the comment below, pass in the new entry to render it, then prepend the element to the DOM so it shows up first: https://developer.mozilla.org/en-US/docs/Web/API/Element/prepend

Let me know if you have questions on this